internal/utils/main.go

```diff
@@ -588,11 +588,18 @@ func (e EnvironmentDetails) CreateCaptiveCoreBackend() (*ledgerbackend.CaptiveSt
 	return backend, err
 }
 
-func (e EnvironmentDetails) GetLedgerCloseMeta(end uint32) (xdr.LedgerCloseMeta, error) {
+func (e EnvironmentDetails) GetUnboundedLedgerCloseMeta(end uint32) (xdr.LedgerCloseMeta, error) {
 	ctx := context.Background()
 
 	backend, err := e.CreateCaptiveCoreBackend()
```

Comment:
I think there is a missing error in `CreateCaptiveCoreBackend`
The err that is returned is only after `backend, err := ledgerbackend.NewCaptive(` but there should be a `if err: return` after `captiveCoreToml, err := ledgerbackend.NewCaptiveCoreTomlFromFile( `

```func (e EnvironmentDetails) CreateCaptiveCoreBackend() (*ledgerbackend.CaptiveStellarCore, error) {
	captiveCoreToml, err := ledgerbackend.NewCaptiveCoreTomlFromFile(
		e.CoreConfig,
		ledgerbackend.CaptiveCoreTomlParams{
			NetworkPassphrase:  e.NetworkPassphrase,
			HistoryArchiveURLs: e.ArchiveURLs,
			Strict:             true,
		},
	)

	if err != nil {
	  return _, err // not sure if the _ will work otherwise probably return a empty or null CaptiveStellarCore
	}

	backend, err := ledgerbackend.NewCaptive(
		ledgerbackend.CaptiveCoreConfig{
			BinaryPath:         e.BinaryPath,
			Toml:               captiveCoreToml,
			NetworkPassphrase:  e.NetworkPassphrase,
			HistoryArchiveURLs: e.ArchiveURLs,
		},
	)
	return backend, err
```

Reply:
Also I think `CreateCaptiveCoreBackend` can probably be used in [operations.go](https://github.com/stellar/stellar-etl/blob/master/internal/input/operations.go#L32) and [transaction.go](https://github.com/stellar/stellar-etl/blob/master/internal/input/transactions.go#L24) cause they just hardcode the same logic right?